test/Makefile

```diff
@@ -103,26 +106,32 @@ run-agbot: agbot-docker-prereqs test-network
 		--add-host "$(DOCKER_AGBOT_ADD_HOST)" \
 		-v /var/run/docker.sock:/var/run/docker.sock \
 		-v $(UDS):$(UDS):rw \
+		-v /var/tmp/horizon/:/var/tmp/horizon/:rw \
```

Comment:
This needs to be /var/tmp/horizon on the host machine?

Reply:
test

Reply:
yes, ess related files like auth and .sock file are put under this directory so that the services can access them.